labjack-ljm/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Version string in the C library installer's file name
+file_version=2019_07_16
+# The desired architecture, as descripbed in the C library installer's file name
+arch=x86_64
+# The version of the C library in the above file.
+# To find this information:
+# * Download and expand the installer.
+# * cd into the new directory.
+# * Run labjack_ljm_installer.run (a self-extracting archive) to expand the library code:
+#   ./labjack_ljm_installer.run --keep --noexec
+# Look in labjack_ljm_software for the library.
+c_version=1.20.1
+# Python wrapper version
+python_version=1.21.0
+
+echo "Download and unpack labjack-ljm C library labjack_ljm_software_${file_version}_${arch}.tar.gz"
+wget https://labjack.com/sites/default/files/software/labjack_ljm_software_${file_version}_${arch}.tar.gz
```

Comment:
If it is Petr to review this, he would suggest to use the `curl` instead of `wget` instead.